.github/workflows/update_goldens.yaml

```
@@ -0,0 +1,46 @@
+      - name: 📦 Install dependencies
+        working-directory: packages/world_flags
+        run: flutter pub get
+
+      - name: ️🧪 Update golden tests
+        working-directory: packages/world_flags
+        run: flutter test --update-goldens
+
+      - name: 🚀 Commit updated golden files
+        uses: stefanzweifel/git-auto-commit-action@v6.0.1
+        with:
+            branch: ${{ inputs.branch }}
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 10 but found 12

(indentation)